### ISSUE TITLE
Nether Quartz and Mystical Agriculture dont stop genning

### ISSUE THREAD

**Issue by BedrockLegends** [waylanderou/almost-all-the-ores#2]
I am using this mod to stop World gen for a 1.15.2 skyblock pack and everything else stops genning but the Prosperity Ore, Inferium Ore, and Vanilla Nether Quartz ore still gen, is there anyway to fix this

**Comment by waylanderou** [comment]
The easiest way to fix Nether Quartz ore generating is waiting for me to post an update with a config option to remove everything in Nether. I will release it by Friday hopefully.

Regarding Mystical Agriculture ores, it's a bit more complicated. I can give you short or long answer, and since I don't know which one you want, I'll give you both:

My mod can remove ores from other mods _only_ when they use a bad way of adding ores to a biome. Forge has a parallel mod loading system, and there is some things you aren't supposed to do in the parallel loading stage. And adding ores to a biome is one of those, because it's not thread-safe and can cause problems. Modders are supposed to tell Forge to run later those risky tasks on the main thread. But surprisingly a lot of mods do not use this safe way and add ores during the parallel loading stage (including me, I need to fix this). Mystical Agriculture is not one of those, and use the safer DeferredWorkQueue. When mods use this queue to add ores, I can't do anything about it, so you'd have to disable manually the ores generation in their config. After looking at their source code, it doesn't seem to be configurable for now. Maybe they'll add it in the future, who knows ?

In short: disable their ores in their config if you can, because AatO can't prevent them from spawning.
